src/js/state/getPersistable.ts

```diff
@@ -24,6 +24,7 @@ export function getWindowPersistable(state: State) {
     delete draft.workspaceStatuses
     delete draft.toolbars
     delete draft.configs
+    delete draft.modal
```

Comment:
I threw this in bc I found a bad RSOD (red screen of death) bug when quitting and starting up again or refreshing the app if a modal is open. There is probably another solution (the error had to do with being unable to locate the modal dom element by it's #id) but I figured that we probably don't care to persist modals anyway.